bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -1,31 +1,93 @@
 import React, { useEffect, useState } from 'react';
-
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 //css
-import  "./css/FilterResults.css"
+import {
+
+  useLocation
+} from "react-router-dom";
+
+import "./css/FilterResults.css"
 
 // core components
+
 import ExamplesNavbar from "../components/Navbars/ExamplesNavbar.js";
 import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
 import List from "../components/List";
 import withListLoading from '../components/withListLoading';
 import DeviceIdentifier from "react-device-identifier";
 
 function ListPage() {
+
   // Consuming REST GET
   const ListLoading = withListLoading(List);
   const [appState, setAppState] = useState({
     loading: false,
-    repos: null,
+    establishments: {},
   });
+
+  var filter = {
+    "filters": {
+    }
+  };
+  //CONSUMING FORM DATA
+  const location = useLocation();
+
   useEffect(() => {
+
     setAppState({ loading: true });
-    const apiUrl = "https://api.github.com/users/hacktivist123/repos";
-    fetch(apiUrl)
-      .then((res) => res.json())
-      .then((repos) => {
-        setAppState({ loading: false, repos: repos });
-      });
-  }, [setAppState]);
+
+    if (location.state != undefined) {
+      var data = location.state[0];
+      console.log(data);
+
+      //MAPPING FORM DATA
+      let beers_aux = [];
+      let zones_aux = [];
+
+      if (data['Paulaner'] == "on") {
+        beers_aux.push("Paulaner");
+      }
+      if (data['Cruzcampo'] == "on") {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7157793)
```suggestion
      if (data["Cruzcampo"] == "on") {
```